frontend/pages/admin/IntegrationsPage/cards/Calendars/Calendars.tsx

```diff
@@ -58,6 +58,16 @@ interface ICalendarsFormData {
   apiKeyJson?: string;
 }
 
+// Used to surface error.message in UI of unknown error type
+type ErrorWithMessage = {
```

Comment:
Nice. This is probably something we can have as a utility type/typeguard for use across the UI.